### PR TITLE
increase readability

### PR DESCRIPTION
**Pull request by FritzOscar**

### CODE REVIEW

**Comment by abstractj** [comment]
@andymunro could you please review when you get the chance?

File: server_admin/topics/sessions/offline.adoc [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 Offline access is a feature described in https://openid.net/specs/openid-connect-core-1_0.html#OfflineAccess[OpenID Connect specification] .
 The idea is that during login, your client application will request an Offline token instead of a classic Refresh token.
-The application can save this offline token in a database or on disk and can use it later even if user is logged out.
-This is useful if your application needs to do some "offline" actions on behalf of user even when the user is not online.
+The application can save this offline token in a database or on disk and can use it later even if the user is logged out.
+This is useful if your application needs to do some "offline" actions on behalf of the user even when she is not online.
```
Comment (andymunro) [inline server_admin/topics/sessions/offline.adoc:8]: Looks good.  However, to make it relatable to men or women, I suggest "that user" in place of "she."

**Review by andymunro — CHANGES_REQUESTED** [review]
Just one minor change. Thanks.

**Comment by abstractj** [comment]
Thanks Andy. @FritzOscar could you please update your PR based on the considerations here?

**Pull request closed without merging**